airbyte-webapp/src/components/base/Input/Input.tsx

```diff
@@ -4,85 +4,16 @@ import classNames from "classnames";
 import React, { useCallback, useRef, useState } from "react";
 import { useIntl } from "react-intl";
 import { useToggle } from "react-use";
-import styled from "styled-components";
-import { Theme } from "theme";
 
 import Button from "../Button";
-
-type IStyleProps = InputProps & { theme: Theme };
-
-const getBackgroundColor = (props: IStyleProps) => {
-  if (props.error) {
-    return props.theme.greyColor10;
-  } else if (props.light) {
-    return props.theme.whiteColor;
-  }
-
-  return props.theme.greyColor0;
-};
+import styles from "./Input.module.scss";
```

Comment:
@benmoriceau are these airbyte-webapp changes intended to be part of this PR? It seems like they are unrelated and maybe the result of a bad merge or something

Reply:
@lmossman It is unrelated. My base branch is not master and every time the base is updated, unrelated changes appears :/. I merge it and the review will display the 6 changes that are actual changes.